test/mainnet/withdrawer.spec.sol

```diff
@@ -0,0 +1,311 @@
+// SPDX-License-Identifier: BSD Clause-3
```

Comment:
main note here is when you finalize ynETH and ynLSDe withdrawals do not attempt to simulate a full proper withdrawal cycle, not needed, too complex

just donate to the ynETHAssetsRedemptionVault and ssetsRedemptionVault respectively (See spec for details)

be careful that when donations are done, the ETH rate of ynETH and ynLSDe increases

Reply:
Yes we are donating right now to the redemption vaults in the setUp of the tests